modules/api-svc/test/__mocks__/@mojaloop/sdk-scheme-adapter-private-shared-lib.js

```diff
@@ -0,0 +1,52 @@
+class KafkaDomainEventConsumer {
```

Comment:
Let me know what you think about moving this to the private-shared-lib so it can be used for other modules?

Reply:
I think this mock is useful only within the api-sdk service. Other services are pretty simple and use DI everywhere.